api_tests/users/views/test_user_actions.py

```diff
@@ -21,6 +21,14 @@ class TestActionFilters(ActionFilterMixin):
     def url(self):
         return '/{}users/me/actions/'.format(API_BASE)
 
+    @pytest.fixture()
+    def expected_actions(self, all_actions, allowed_providers):
+        expected = super(TestActionFilters, self).expected_actions(all_actions, allowed_providers)
+        node = expected[0].target.node
+        node.is_public = False
+        node.save()
+        return [a for a in expected if a.target.node.is_public]
+
     def test_no_permission(self, app, url, expected_actions):
```

Comment:
What was `expected_actions` before? (I can't seem to figure out where is came from)

Reply:
It's a pytest fixture [defined on the parent class, `ActionFilterMixin`](https://github.com/aaxelb/osf.io/blob/f549e490357e0b5052e75408acbf7e03eb851aed/api_tests/reviews/mixins/filter_mixins.py#L69). It's the subset of actions that should be visible to the user in the api. The overriding method calls `super()` and filters it further to exclude actions on preprints with private nodes.